### DISCUSSION TITLE
New Tabs

### DISCUSSION THREAD

**Discussion by raphhawk** [helix-editor/helix#4271] [Q&A] [answered]
Hi All, is there a way to get the :tabnew functionality of vim in helix? How to disable the ": " suggestion box that appear int the bottom.

**Comment by the-mikedavis** [comment, marked as answer]
Tabs are not yet implemented: https://github.com/helix-editor/helix/issues/2295

**Comment by Omnikar** [comment]
If you add `bufferline = "multiple"` or `bufferline = "always"` in your config then all open buffers will be displayed like tabs.
